pkg/cmd/workspace/logs.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package workspace
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/daytonaio/daytona/cmd/daytona/config"
+	"github.com/daytonaio/daytona/internal/util"
+	apiclient_util "github.com/daytonaio/daytona/internal/util/apiclient"
+	logs_view "github.com/daytonaio/daytona/pkg/views/logs"
+	"github.com/daytonaio/daytona/pkg/views/workspace/selection"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+var followFlag bool
+var retryFlag bool
+
+var LogsCmd = &cobra.Command{
+	Use:     "logs [WORKSPACE] [PROJECT]",
+	Short:   "View logs for a workspace or project",
+	GroupID: util.WORKSPACE_GROUP,
+	Args:    cobra.RangeArgs(0, 2),
+	Aliases: []string{"log"},
+	Run: func(cmd *cobra.Command, args []string) {
+		c, err := config.GetConfig()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		activeProfile, err := c.GetActiveProfile()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		query := ""
+		if retryFlag && followFlag {
+			query += "follow=true&retry=true"
+		} else if retryFlag {
+			query += "retry=true"
+		} else if followFlag {
+			query += "follow=true"
+		}
+
+		ctx := context.Background()
+		var workspaceId string
+		var projectName string
+
+		apiClient, err := apiclient_util.GetApiClient(&activeProfile)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		if len(args) == 0 {
```

Comment:
If the user does not provide an argument, they should be presented with a workspace select and then a project select (with the option to get only the workspace logs).

Reply:
added -w flag to just get workspace logs.